dashboard/app/views/ops_mailer/workshop_reminder.html.haml

```diff
@@ -5,7 +5,7 @@
 %p
   As part of your commitment to this program, you are expected to participate in this workshop. If you have any conflicts with these dates, please email
   %a{href:'mailto:pd@code.org'}
```

Comment:
could use the `mail_to` helper here and it would be more compact and not have to write the email twice: http://apidock.com/rails/ActionView/Helpers/UrlHelper/mail_to